src/main/java/org/folio/rest/impl/DataImportImpl.java

```diff
@@ -434,7 +434,7 @@ public void getDataImportUploadUrl(String fileName, Map<String, String> okapiHea
     vertxContext.runOnContext(v -> {
       try {
         LOGGER.debug("getDataImportUploadUrl:: getting upload url");
```

Comment:
Could you add some additional details to logs like a fileName or partNumber for making the support process in the future more clear? Thanks